js/my_first_tour.js

```diff
@@ -0,0 +1,75 @@
+// Define the tour!
+var tour = {
+  id: "hello-hopscotch",
+  steps: [
+    {
+      title: "Navigation improvements!",
+      content: "We've improved the navigation for the Docker documentation!<br /><br />This short tour takes less than a minute. It highlights the changes, then returns you to the current page. You'll learn about:<br /><ul><li>New top navigation</li><li>Left-hand navigation</li><li>Feedback links</li><li>In-page navigation</li></ul>",
+      target: "main-content",
+      placement: "top",
+      xOffset: "center",
+      yOffset: "300px",
+      onShow: function () {
+        // Hide the arrow on the first navigation bubble
+        $('.hopscotch-bubble-arrow-container').css('visibility', 'hidden');
+      }
+    },
+    {
+      title: "Top Navigation",
+      content: "Use the top navigation to discover the types of content available.",
+      target: "top-nav",
+      placement: "bottom",
+      onShow: function () {
+        // Show the arrow again
+        $('.hopscotch-bubble-arrow-container').css('visibility', 'visible');
+      }
+    },
+    {
+      title: "Left Navigation",
+      content: "Use the left navigation for a structured view of content within a top-level category.",
+      target: "left-nav",
+      placement: "right"
+    },
+    {
+      title: "Feedback Links",
+      content: "Use the feedback links to edit the page, provide feedback, or find out how to get support.",
+      target: "feedback-links",
+      placement: "left",
+      multipage: "true",
```

Comment:
For these three lines, I really only need to do it this way if the page they start the tour on doesn't have the div `#side-toc`. But I'm not sure how to do that without duplicating the whole `var tour` with two variants. @jsouth or @johndmulhausen do you have any suggestions?